drivers/backup/portworx/portworx.go

```diff
@@ -0,0 +1,232 @@
+	refreshEndpoint bool
+	token           string
+}
+
+func (p *portworx) String() string {
+	return driverName
+}
+
+func (p *portworx) Init(schedulerDriverName string, nodeDriverName string, volumeDriverName string, token string) error {
+	var err error
+
+	logrus.Infof("using portworx backup driver under scheduler: %v", schedulerDriverName)
+
+	p.nodeDriver, err = node.Get(nodeDriverName)
+	if err != nil {
+		return err
+	}
+	p.token = token
+
+	p.schedulerDriver, err = scheduler.Get(schedulerDriverName)
+	if err != nil {
+		return fmt.Errorf("Error getting scheduler driver %v: %v", schedulerDriverName, err)
+	}
+
+	p.volumeDriver, err = volume.Get(volumeDriverName)
+	if err != nil {
+		return fmt.Errorf("Error getting volume driver %v: %v", volumeDriverName, err)
+	}
+
+	if err = p.setDriver(pxbServiceName, pxbNamespace); err != nil {
+		return fmt.Errorf("Error setting px-backup endpoint: %v", err)
+	}
+
+	return err
+
+}
+
+func (p *portworx) constructURL(ip string) string {
+	return fmt.Sprintf("%s:%d", ip, defaultPxbServicePort)
+}
+
+func (p *portworx) testAndSetEndpoint(endpoint string) error {
+	pxEndpoint := p.constructURL(endpoint)
+	conn, err := grpc.Dial(pxEndpoint, grpc.WithInsecure())
+	if err != nil {
+		logrus.Errorf("unable to get grpc connection: %v", err)
+		return err
+	}
+
+	p.healthManager = api.NewHealthClient(conn)
+	_, err = p.healthManager.Status(context.Background(), &api.HealthStatusRequest{})
+	if err != nil {
+		logrus.Errorf("HealthManager API error: %v", err)
+		return err
+	}
+
+	p.clusterManager = api.NewClusterClient(conn)
+	p.backupLocationManager = api.NewBackupLocationClient(conn)
+	p.cloudCredentialManager = api.NewCloudCredentialClient(conn)
+	p.backupManager = api.NewBackupClient(conn)
+	p.restoreManager = api.NewRestoreClient(conn)
+	p.backupScheduleManager = api.NewBackupScheduleClient(conn)
+	p.schedulePolicyManager = api.NewSchedulePolicyClient(conn)
+	p.organizationManager = api.NewOrganizationClient(conn)
+
+	logrus.Infof("Using %v as endpoint for portworx backup driver", pxEndpoint)
+
+	return err
+}
+
+func (p *portworx) GetServiceEndpoint(serviceName string, nameSpace string) (string, error) {
+	svc, err := core.Instance().GetService(serviceName, nameSpace)
+	if err == nil {
+		return svc.Spec.ClusterIP, nil
+	}
+	return "", err
+}
+
+func (p *portworx) setDriver(serviceName string, nameSpace string) error {
+	var err error
+	var endpoint string
+
+	endpoint, err = p.GetServiceEndpoint(serviceName, nameSpace)
+	if err == nil && endpoint != "" {
+		if err = p.testAndSetEndpoint(endpoint); err == nil {
+			return nil
+		}
+	} else if err != nil && len(node.GetWorkerNodes()) == 0 {
+		return err
+	}
+	return fmt.Errorf("failed to get endpoint for portworx backup driver: %v", err)
+}
+
+func (p *portworx) CreateOrganization(req *api.OrganizationCreateRequest) (*api.OrganizationCreateResponse, error) {
+	org := p.organizationManager
+	return org.Create(context.Background(), req)
```

Comment:
```suggestion
	return p.organizationManager.Create(context.Background(), req)
```
Same for all the other APIs

Reply:
fixed here https://github.com/portworx/torpedo/pull/429/commits/a8b35120d61aefff4e5ca6e662258a7230f627be